packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -196,7 +196,7 @@ class CallContext {
       sounds: options?.callingSounds,
       reactions: options?.reactionResources
     };
-    this.emitter.setMaxListeners(options?.maxListeners ?? 50);
+    this.emitter.setMaxListeners(options?.maxListeners ?? 125);
```

Comment:
Add a comment here about this number, possibly make a constant at the top of the file
```
// Something about how each use of useSelector counts as 1 of these
// Something else about how this will need to be increased with usage of `useSelector`
const EVENT_EMITTER_WARNING_LIMIT = 125;
```